docs/src/main/paradox/scala/http/routing-dsl/directives/parameter-directives/parameters.md

```diff
@@ -55,7 +55,10 @@ as required, optional, or repeated, or to filter requests where a parameter has
 You can use @ref[Case Class Extraction](../../case-class-extraction.md#case-class-extraction) to group several extracted values together into a case-class
 instance.
 
-Requests missing a required parameter or parameter value will be rejected with an appropriate rejection.
+Requests missing a required parameter or parameter value will be rejected with an appropriate rejection. 
+
+If an unmarshaller throws an exception while extracting the value of a parameter, it will be handled as a rejection.
```

Comment:
> Do these names change often?

The rejection names? No, they're pretty set in stone at this point, technically they could change but due to compatibility reasons we aim no to change them